### ISSUE TITLE
Use method missing instead of just call?

### ISSUE THREAD

**Issue by phuongnd08** [soveran/nest#8]
I think the old interface is what make nest great. Instead of sterilze this to only "call" I think we'd better use method missing instead. The syntax error still propagate through RuntimeError exception What do you say?

**Comment by soveran** [comment]
@phuongnd08 What if we define `method_missing` in addition to having `call`? I can push a version to a branch so we can discuss it.

**Comment by soveran** [comment]
I have just tested this:

```ruby
# Operating with Redis with dynamic methods.
scope do
  prepare do
    @redis = Redic.new
    @redis.call("FLUSHDB")

    class Nest
      def method_missing(m, *args)
        call(m, *args)
      end
    end
  end

  test "relay missing methods as Redis commands" do
    n1 = Nest.new("foo")
    n1.set("s1")

    assert "s1" == n1.get
  end
end
```

**Comment by phuongnd08** [comment]
This looks great!

**Comment by phuongnd08** [comment]
Could you release this?

**Comment by soveran** [comment]
@phuongnd08 Sorry! I had forgotten to release this change. I've just released version 3.1.0, which includes this feature.
